helm/idol-library-example-test/values.yaml

```
@@ -125,6 +125,19 @@ ingress:
   # e.g. https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations
   annotations: {}
 
+  TLS:
```

```suggestion
  tls:
```
Think we should stick to camelCase variable naming (also closer to the kubernetes spec - [example](https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/service/networking/tls-example-ingress.yaml))